src/lib/primitives/jsonschema/jsonschema.ts

```diff
@@ -123,6 +124,9 @@ const schemaWalker: JsonSchemaWalker = {
                 (isBoolean(node.additionalProperties) && keysOf(node.additionalProperties).length === 0)) &&
```

Comment:
<i class='graphite__hidden'>[Re: lines 124 to 124]<br/></i>

don't we have a clash between `isBoolean` and `keysOf` here? :thinking: I would expect a negation here at first glance (or maybe I need a coffee first)<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/skyleague/therefore/56?utm_source=unchanged-line-comment">Graphite</a>.</span>